openfisca_core/tests/assets/parameters.yaml

```diff
@@ -0,0 +1,124 @@
+
+# Regular parameter values : meta + VALUES handling variation of its value in time
+- variable: smic_horaire_brut
+  description: SMIC horaire brut
+  format: float
+  type: monetary
+  VALUES:
+    - {deb: 2016-01-01, fuzzy: true,     valeur: 9.67}
+    - {deb: 2015-01-01, fin: 2015-12-31, valeur: 9.61}
+    - {deb: 2014-01-01, fin: 2014-12-31, valeur: 9.53}
+    - {deb: 2013-01-01, fin: 2013-12-31, valeur: 9.43}
+    # etc.
+
+#  Short date notation :
+#  Use it in the very common case where all VALUE leaves are consecutive
+- variable: plafond_securite_sociale
+  description: Plafond mensuel de la sécurité sociale
+  type: monetary
+  # Values should be written in descending ordered :
+  # what's most important today first
+  VALUES:
+    #  Fuzzy is explicit :
+    #  to express the disparition of a variable,
+    #  you should explicitely set its last date
+    #  (which is the first in the syntax) to 0
+    2017-01-01: 3218
+    # This first line above is computationally useless.
+    # But to humans with a limited capacity for watching parameter changes,
+    # it ensures that somebody checked the law and the value had not changed in 2017.
+    2016-01-01: 3218
+    2015-01-01: 3170
+    # Asking for a parameter for the date before the last will
+    # raise a ParameterNotFound error
+    # TODO: Or should it return 0 ?
+
+
+
+# BAREME node with 1 bracket...
+- variable: famille
+  description: Allocations familiales
+  BAREME:
+    - SEUIL:
+        VALUES:
+          2015-01-01: 0
+          1992-01-01: 0
+      TAUX:
+        VALUES:
+          2014-01-01: .0525
+          1993-07-01: .054
+
+
+# ... which is annoying for this regular pattern of "base * rate"
+# TODO implement this !
+
+- variable: famille2
+  description: Allocations familiales
+  LIN:
+    # ASSIETTE: assiette_cotisations_sociales # TODO: retrieve this var or request it ?
+    VALUES:
+      2018-01-01: .019
+      2017-01-01: .019
+      2016-01-01: .0185
+
+# LIN with PLAFOND can be introduced too:
+
+- variable: vieillesse_deplafonnee_salarie
+  description: Cotisation déplafonnée de vieillesse (salarié)
+  LIN:
```

Comment:
Yes, with an option for applying the rate to a *base* possibly limited by a ceiling multiplied by a *factor*. At the moment, the base and ceiling factor are given to the parameter call.
LIN is a code-name, it could be LINEAR or anything else.